src/App.tsx

```diff
@@ -15,49 +15,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+export function getReorderedGoods(
```

Comment:
that moment when you export a function, but your entire app is one file 😆

Reply:
JS does have garbage collection, but I don't :|